src/adv.py

```diff
@@ -137,7 +141,19 @@
                         elif player.current_room.items[3].name == i:
                             del player.current_room.items[3]
                             print("\nRemoved " + str(i) + " from " + str(player.current_room.name) + "")
-
+            elif select == "d":
+                for i in player.stash:
+                    dropchoice = input("Are you sure you want to drop " + i + " in " + "? y/n : ")
+                    if dropchoice == "y":
+                        player.current_room.set_items([Item(i, "Dropped item")])
+                        player.stash.remove(i)
+                    elif dropchoice == "n":
+                        print("Declined to drop " + i)
+                    else:
+                        print("Invalid input")
+                        pass
+            else:
+                pass 
         else:
             print('Invalid input. Please try again.\n')
     except ValueError:
```

Comment:
After looking through this code, you all went ham on styling and trying to hit MVP, please avoid this future. Prioritize MVP and then Stretch ( while doing stretches do refactor)